dev_build/docker-compose.yml

```diff
@@ -5,13 +5,26 @@ services:
     networks:
       app_net:
     hostname: pyglidein-server
+    volumes:
+      - "/home/hskarlupka/code/pyglidein/tests:/pyglidein/tests"
```

Comment:
These volumes seem hard-coded to your machine.  Is there  a better way to do this?  Maybe a relative path, or something that builds this docker-compose file on the fly?

Reply:
@dsschult Good call.  I checked out the docker-compose docs and it looks like you can use relative paths.  For now I'm going to volume in the entire pygliden directory on the client and server for easy and quick dev testing.  Once we move to python packages we could think of voluming something else.